tests/test_optimizer_dryruns.py

```diff
@@ -468,8 +468,8 @@ def test_invalid_image(monkeypatch):
     assert 'Cloud must be specified' in str(e.value)
```

Comment:
Should we add a smoke test for the image_id for Azure as well?

Reply:
Added and passed. Thanks!